examples/connatix-player.amp.html

```diff
@@ -39,6 +40,13 @@ <h3>Connatix Player Responsive - With Media Id</h3>
     width="480"
     height="270">
   </amp-connatix-player>
-
+  <h3>Connatix Player With Docking</h3>
```

Comment:
revert the changes in this file for now. looks like this is the issue blocking us.